setup.cfg

```diff
@@ -19,7 +19,7 @@ classifiers =
     Development Status :: 3 - Alpha
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
-    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
```

Comment:
Nice catch, never realized that we had mismatched versions in this specification and the testing!